_gsocproposals/2020/proposal_VecGeomGJK.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Implementation of Gilbert-Johnson-Keerthi Algorithm for Convex Shapes in VecGeom
+layout: gsoc_proposal
+project: VecGeom
+year: 2020
+organization:
+    - CERN
+---
+
+## Description
+
+The VecGeom [1] project aims at developing a high-performance library for
```

Comment:
I would rather drop the article-like reference and put a markdown link directly on the keyword (here VecGeom). This makes it easier for the reader to directly go there if they want. I would keep also the links in the "Links" section, but as a bullet list.

Reply:
I copied this from a previous project, but I agree.